wordpress/wordpress/Dockerfile

```diff
@@ -0,0 +1,122 @@
+# =============================================================================
+# wordpress:5-php7.3-apache
+#
+# WordPress 5.*, Apache 2.4, PHP 7.3
+# Modification: Rsyslog/Fail2Ban setup
+#
+# =============================================================================
+FROM wordpress:5-php7.3-apache
```

Comment:
There is a php7.4 version out now, update this.
https://hub.docker.com/_/wordpress

Reply:
From @sjbruce Check if conflicts with current plugins on dev first before rolling out.